client/package.json

```diff
@@ -121,12 +123,21 @@
     "validator": "13.6.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.13.16",
+    "@storybook/addon-actions": "^6.2.9",
+    "@storybook/addon-essentials": "^6.2.9",
+    "@storybook/addon-links": "^6.2.9",
+    "@storybook/addon-postcss": "^2.0.0",
+    "@storybook/react": "^6.2.9",
     "@testing-library/jest-dom": "5.12.0",
     "@testing-library/react": "11.2.6",
     "autoprefixer": "9.8.6",
+    "babel-loader": "^8.2.2",
     "babel-plugin-transform-imports": "1.5.1",
     "chokidar": "3.5.1",
     "copy-webpack-plugin": "5.1.2",
+    "dotenv-webpack": "^7.0.2",
+    "html-webpack-plugin": "^5.3.1",
```

Comment:
If I understand https://github.com/storybookjs/storybook/issues/14403 correctly, Storybook is using a very old version of these libraries. They are hoisted and got picked up by `webpack`, but we got error because they are not compatible with webpack 5. 

Apparently Storybook team fixed it but then reverted the PR due to it being backward incompatible (https://github.com/storybookjs/storybook/issues/14403#issuecomment-817138766). I followed https://github.com/storybookjs/storybook/issues/14497#issuecomment-818838975 and added these packages to our dev dependency list as a workaround.

Reply:
The Storybook team recommends using Yarn's `resolutions` to workaround this issue. It is not applicable to us, but I'm just posting the link here for future reference: https://gist.github.com/shilman/8856ea1786dcd247139b47b270912324#yarn-resolutions